src/content.config.ts

```diff
@@ -673,6 +673,18 @@ const footer = defineCollection({
   ),
 });
 
+const notFoundPage = defineCollection({
+  loader: collectionLoader("globals/404"),
```

Comment:
Update to "globals/not-found-page" since we had to change from `404` due to type generation error in https://github.com/cloud-gov/pages-editor/pull/207.